createdisk.sh

```diff
@@ -156,9 +176,15 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crictl pods -q)'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl pods -q)'
 
-# Get rhcos ostree Hash ID
+# Get the rhcos ostree Hash ID
```

Comment:
This belongs to the previous commit, but I'd just squash these 2 commits together.